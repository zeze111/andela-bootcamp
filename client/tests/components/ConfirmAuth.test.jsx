import React from 'react';

import Footer from '../../components/Footer';
import Authorize from '../../components/confirmAuth';
import store from '../../store';
import mockLocalStorage from '../mocks/localStorage';


window.localStorage = mockLocalStorage;

const components = Authorize(Footer);

const {WrappedComponent} = components;

const props = {
  signout: jest.fn(),
  auth: { }
};

const context = {
  router: {
    history: {
      push: jest.fn()
    }
  }
};

window.localStorage.setItem('jwtToken', 'kdrhkhdkjhvzhdjvhvbd.jvgjhxbdvh.jdvcxhjvxhvvâˆ«');

describe('Test for Confirm Authentication', () => {
  it('should render component', () => {
    const wrapper = shallow(<WrappedComponent {...props}/>, { context } );
    expect(wrapper).toMatchSnapshot()
  });
});
