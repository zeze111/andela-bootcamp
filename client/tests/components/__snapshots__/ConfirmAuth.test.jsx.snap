// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Confirm Authentication should render component 1`] = `
<Footer
  auth={Object {}}
  signout={
    [MockFunction] {
      "calls": Array [
        Array [],
      ],
    }
  }
/>
`;
